components/BarList/index.js

```diff
@@ -8,10 +8,11 @@ const matchesInBar = barsWithMatches.map((bar) => (
 {...bar, 
 matches: matches.filter((match) => bar.matches.includes(match.id)).map((team) => ({homeTeam: team.homeTeam, awayTeam: team.     awayTeam})) 
 }))
+
     return(
         <BarsUl>
```

Comment:
For the naming of your styled components: We prefer to write full names and do not use the abbreviated HTML element names. In this case I would say you could just go with `List` as name as your component already has the name `BarsList`. 🤔

Reply:
done :-)